.gitignore

```diff
@@ -4,3 +4,4 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
+.idea/
```

Comment:
That's fine—just omit it then. I have no emotional attachment to this particular change!